### ISSUE TITLE
Automate VOC updates

### ISSUE THREAD

**Issue by talnor** [Clinical-Genomics/MUTANT#118]
**What needs to be done:**
VOCs and spike mutations should always reflect the latest definitions by ECDC. 

Definitions are available here: 
* https://www.ecdc.europa.eu/en/covid-19/variants-concern

**Suggestions on how to get it done:**
TBD. Small parsing script written already. Can we run it in github actions maybe?

**What are the arguments for getting it done:**

**Task is considered finished when:**
- VOCs and spike mutations are updated on a regular basis automatically.

**Comment by sofstam** [comment]
@talnor good idea! We can test to run the parsing script with a cron job and gh action

**Comment by henrikstranneheim** [comment]
@sofstam We are trying to move away from cron jobs and use systemd services instead. See, https://atlas.scilifelab.se/infrastructure/development/development/#systemd,  http://systemd.io/ and https://en.wikipedia.org/wiki/Systemd

**Comment by sofstam** [comment]
@henrikstranneheim Thank you for the clarification :)
